src/components/Moveable.vue

```diff
@@ -44,14 +44,15 @@ export default {
     throttleScale: Number,
     throttleRotate: Number,
     keepRatio: Boolean,
+    parent: Object
```

Comment:
It must be called `container`:
https://github.com/daybrush/moveable/blob/master/src/Moveable.tsx#L48